src/main/java/org/jabref/gui/util/ThemeLoader.java

```diff
@@ -21,28 +23,39 @@
 
 /**
  * Installs the style file and provides live reloading.
- *
+ * <p>
  * The live reloading has to be turned on by setting the <code>-Djabref.theme.css</code> property.
  * There two possible modes:
- *  (1) When only <code>-Djabref.theme.css</code> is specified, then the standard <code>Base.css</code> that is found will be watched
- *      and on changes in that file, the style-sheet will be reloaded and changes are immediately visible.
- *  (2) When a path to a css file is passed to <code>-Djabref.theme.css</code>, then the given style is loaded in addition to the base css file.
- *      Changes in the specified css file lead to an immediate redraw of the interface.
- *
+ * (1) When only <code>-Djabref.theme.css</code> is specified, then the standard <code>Base.css</code> that is found will be watched
+ * and on changes in that file, the style-sheet will be reloaded and changes are immediately visible.
+ * (2) When a path to a css file is passed to <code>-Djabref.theme.css</code>, then the given style is loaded in addition to the base css file.
+ * Changes in the specified css file lead to an immediate redraw of the interface.
+ * <p>
  * When working from an IDE, this usually means that the <code>Base.css</code> is located in the build folder.
  * To use the css-file that is located in the sources directly, the full path can be given as value for the "VM option":
  * <code>-Djabref.theme.css="/path/to/src/Base.css"</code>
- *
  */
 public class ThemeLoader {
 
     private static final String DEFAULT_PATH_MAIN_CSS = JabRefFrame.class.getResource("Base.css").toExternalForm();
-    private static final String CSS_SYSTEM_PROPERTY = System.getProperty("jabref.theme.css");
     private static final Logger LOGGER = LoggerFactory.getLogger(ThemeLoader.class);
+    private final Path CSS_SYSTEM_PROPERTY;
     private final FileUpdateMonitor fileUpdateMonitor;
 
-    public ThemeLoader(FileUpdateMonitor fileUpdateMonitor) {
+    public ThemeLoader(FileUpdateMonitor fileUpdateMonitor, JabRefPreferences jabRefPreferences) throws JabRefException {
         this.fileUpdateMonitor = Objects.requireNonNull(fileUpdateMonitor);
+
+        String cssFileName = jabRefPreferences.get(JabRefPreferences.FX_THEME);
```

Comment:
First and foremost `CSS_SYSTEM_PROPERTY` should be `System.getProperty("jabref.theme.css")`, if the latter is not empty /null (StringUtil.isBlank) so that one can override the theme from the preference using command line arguments (mainly for developers). The preference should be consulted only if the system property is empty.
Please also rename `CSS_SYSTEM_PROPERTY` to `userCss` or something like this.